code/drasil-gen/lib/Language/Drasil/Generate.hs

```diff
@@ -89,12 +105,27 @@ prntCSS docType fn body = do
   where
     getFD dtype = show dtype ++ "/HTML/"
 
--- | Renders the documents.
+-- | Helper for generating the .toml config file for mdBook.
+printBook :: DocType -> Document -> PrintingInformation -> IO()
+printBook dt doc sm = do
```

Comment:
Why `printBook` instead of `prntBook` (without an "i") for consistency with other functions? Or, better yet, why is `prnt` used in other functions?

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3823/commits/f7ba8fda90132e900e5349385606160f9c09ac72.